### PR TITLE
Use generic names that describe purpose in tests.

### LINKED ISSUE #294: Replace OSL-specific names with generic names.
As it is, our tests contain very OSL-specific names, with users such as `tschuy` and `patcht` and projects such as `pgd` and `gwm`. Replace these names with more generic ones, both for professionalism and ease of reading by non-OSL users/developers.

### PR DESCRIPTION
**Pull request by LyonesGamer**
Fixes #294.

## Changes in this PR.

- [X] Replace usernames with more generic ones describing their roles
- [X] Replace projects with more generic ones describing their forms

@osuosl/devs

### CODE REVIEW

**Comment by ElijahCaine** [comment]
This is a very welcomed change. I'd like to see us do this for all future projects.

@LyonesGamer Have you considered changing the example code-blocks in the docs as well? If you don't think that's necessary this gets a :+1: from me.

**Comment by ElijahCaine** [comment]
I think those example I was remembering were in osuosl/timesync repo so there's a lot less to change than I expected.

:+1: from me.

File: tests/activities.js [hunk 1]
```diff
@@ -506,7 +506,7 @@ module.exports = function(expect, request, baseUrl) {
     });
 
     it('successfully creates a new activity by a manager', function(done) {
-      getAPIToken('patcht', 'drowssap').then(function(token) {
+      getAPIToken('sManager', 'drowssap').then(function(token) {
```
Comment (Kennric) [inline tests/activities.js]: Site_Manager would read better than sManager, I think

**Comment by Kennric** [comment]
This looks pretty good, though I would prefer more verbose and descriptive names. Not critical, but a find/replace would be nice.

**Pull request merged**